docker/build_helper.sh

```diff
@@ -25,12 +25,18 @@ shift $((OPTIND - 1))
 export ARCH=${ARCH}
 export CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-Release}
 export BUILD_TEST=${BUILD_TEST:-ON}
+export BUILD_DALI_KERNELS=${BUILD_DALI_KERNELS:-ON}
+export BUILD_DALI_PIPELINE=${BUILD_DALI_PIPELINE:-ON}
+export BUILD_DALI_OPERATORS=${BUILD_DALI_OPERATORS:-ON}
 export BUILD_BENCHMARK=${BUILD_BENCHMARK:-ON}
 # use a default value as it differs for CUDA 9.x and CUDA 10.x
 export BUILD_NVTX=${BUILD_NVTX}
 export BUILD_PYTHON=${BUILD_PYTHON:-ON}
 export BUILD_LMDB=${BUILD_LMDB:-ON}
 export BUILD_JPEG_TURBO=${BUILD_JPEG_TURBO:-ON}
+export BUILD_FFMPEG=${BUILD_FFMPEG:-ON}
```

Comment:
remove, BUILD_FFMPEG is an internal variable (BUILD_NVDEC is the exposed option)

Reply:
Done